### PR TITLE
Add winery to the setup

### PR DESCRIPTION
**Pull request by mar-be**
Quantil uses Winery as a QAA repository

<s>To do: 

- [ ] set version of qc-atlas </s>

Signed-off-by: Marvin Bechtold <marvin.bechtold.dev@gmail.com>

### CODE REVIEW

**Review by salmma — CHANGES_REQUESTED** [review]
Please commit into 'latest' not 'master'. 

(As soon as the UI is ready, also update all three components in master to their latest release versions.)

File: docker-compose.yml [hunk 1]
```diff
@@ -51,17 +51,34 @@ services:
...
   qc-atlas:
-    image: planqk/atlas:v3.1.1
+    image: planqk/atlas:latest
     environment:
       POSTGRES_HOSTNAME: db
       POSTGRES_PORT: 5060
       POSTGRES_USER: planqk
       POSTGRES_PASSWORD: planqk
       POSTGRES_DB: planqk
+      WINERY_HOSTNAME: host.docker.internal
```
Comment (lharzenetter) [inline docker-compose.yml]: In this case, also `winery` works as Docker resolves the service name

File: docker-compose.yml [hunk 2]
```diff
@@ -51,17 +51,34 @@ services:
...
+    image: opentosca/winery:latest
+    profiles: ["all", "winery"]
+    environment:
+      WINERY_PORT: 8091
+      WORKFLOWMODELER_PORT: 8091
+      TOPOLOGYMODELER_PORT: 8091
+      WINERY_HOSTNAME: host.docker.internal
+      WORKFLOWMODELER_HOSTNAME: host.docker.internal
+      TOPOLOGYMODELER_HOSTNAME: host.docker.internal
+      CONTAINER_HOSTNAME: host.docker.internal
```
Comment (lharzenetter) [inline docker-compose.yml]: You can commit that because the container is not runnig

File: docker-compose.yml [hunk 3]
```diff
@@ -51,17 +51,34 @@ services:
...
     networks:
       - default
+  winery:
+    image: opentosca/winery:latest
+    profiles: ["all", "winery"]
+    environment:
+      WINERY_PORT: 8091
+      WORKFLOWMODELER_PORT: 8091
+      TOPOLOGYMODELER_PORT: 8091
+      WINERY_HOSTNAME: host.docker.internal
```
Comment (lharzenetter) [inline docker-compose.yml]: I'm not quite sure if this works correctly. Do we have a `.env`-file with a hostname? Maybe we should use this instead.

Comment (mar-be) [reply docker-compose.yml]: I added a .env file

File: docker-compose.yml (hunk 1 above)
Comment (mar-be) [reply docker-compose.yml]: ` winery ` does not work for me here

**Review by lharzenetter — APPROVED** [review]

**Review by salmma — APPROVED** [review]

**Pull request merged**